lib/index.js

```diff
@@ -63,7 +77,7 @@ function makeTask (modulePath, file) {
     const where = path.join(modulePath, file)
     fs.stat(where, (err, stats) => {
       if (err && err.code === 'ENOENT') return cb(null, false)
-      if (err) throw err
```

Comment:
As discussed in the ticket, throwing inside of an async function won't make the Promise forward the error so that the user can use `catch()`. We also need to be vary of not throwing since we support a callback api as well (and that won't get automatically called as promises do because of their implicit try-catch).